.github/dependabot.yml

```
@@ -0,0 +1,22 @@
+version: 2
+updates:
+- package-ecosystem: gomod
+  directory: "/"
+  schedule:
+    interval: weekly
```

I would change this to daily
```suggestion
    interval: daily
```